packages/manager/apps/pci-instances/src/data/hooks/instance/polling/useInstancesPolling.ts

```diff
@@ -72,7 +72,7 @@ export const useInstancesPolling = (
     combine: useCallback(
       (results: UseQueryResult<TInstance>[]) =>
         results.map(({ data, error, isLoading }, index) => ({
-          id: pendingTasks[index].instanceId,
+          id: pendingTasks[index]?.instanceId ?? '',
```

Comment:
```suggestion
          id: pendingTasks[index]!.instanceId,
```
Since results and pendingTasks are the same size it will always give a result